landing_page/app/controllers/application_controller.rb

```diff
@@ -1,21 +1,19 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
 
-   protect_from_forgery with: :exception
+  add_flash_types :danger, :info, :warning, :success
 
-   add_flash_types :danger, :info, :warning, :success
+  include SessionsHelper
 
-   include SessionsHelper
-
-   rescue_from CanCan::AccessDenied do |exception|  
-      flash[:danger] = "Access denied!"  
-      redirect_to login_path 
-    end  
-
-    before_action :set_cache_buster
-    def set_cache_buster
-      response.headers["Cache-Control"] = "no-cache, no-store, max-age=0, must-revalidate"
-      response.headers["Pragma"] = "no-cache"
-      response.headers["Expires"] = "Fri, 01 Jan 1990 00:00:00 GMT"
-    end
+  rescue_from CanCan::AccessDenied do |_exception|
```

Comment:
Currently you're never using this `_exception` variable, so it may be a good idea to see if it can be excluded, or just log the error to the rails console for debugging purposes?

Reply:
This exception occurs if someone tries to access someone's user profile without being logged in

Reply:
I more meant that the variable itself is never being used, rather than the exception will never occur.

Reply:
Ah I see, I've removed it and it still works so thanks for spotting it 👍

Reply:
commit 00226e8f6aa8d22925c72076234aa5830f0f7cae